pkg/js/helpers.js

```diff
@@ -294,6 +286,9 @@ function NO_PURGE(d) {
   d.KeepUnknown = true
 }
 
+/**
+ * @deprecated
```

Comment:
I'm ok deleting "deprecated code" in this file. Nobody should be depending on the internal working things like this function.